lib/trino-orc/src/test/java/io/trino/orc/OrcTester.java

```diff
@@ -964,10 +970,10 @@ public static DataSize writeOrcColumnHive(File outputFile, Format format, Compre
             throws Exception
     {
         RecordWriter recordWriter = createOrcRecordWriter(outputFile, format, compression, type);
-        return writeOrcFileColumnHive(outputFile, recordWriter, type, values);
+        return writeOrcColumnHive(outputFile, recordWriter, type, values);
     }
 
-    public static DataSize writeOrcFileColumnHive(File outputFile, RecordWriter recordWriter, Type type, Iterator<?> values)
+    public static DataSize writeOrcColumnHive(File outputFile, RecordWriter recordWriter, Type type, Iterator<?> values)
```

Comment:
pre-existing: `outputFile` does not need to be passed to this method.

`succinctBytes(outputFile.length())` can be calculated after the `writeOrcColumnHive` call.

Reply:
`writeOrcColumnHive` methods atm are not changing due to https://github.com/trinodb/trino/pull/15434#discussion_r1053033720 so make sense to keep them as before.

Reply:
done